static/app/components/comboBox/index.tsx

```diff
@@ -242,15 +242,17 @@ function ControlledComboBox<Value extends string>({
 
   // Sync input value with value prop
   const previousValue = useRef(value);
-  if (previousValue.current !== value) {
-    const selectedLabel = options
-      .flatMap(item => ('options' in item ? item.options : [item]))
-      .find(option => option.value === value)?.label;
-    if (selectedLabel) {
-      setInputValue(selectedLabel);
+  useEffect(() => {
```

Comment:
is this `useEffect` needed? I'm not very familiar with this code but I assume that we are storing value in the ref in order to avoid a re-render every time the value changes.

Reply:
But I have missed to remove `useRef` here, now that is fixed